### DISCUSSION TITLE
Error: Invalid Input Error: Malformed parquet file When read_parquet from S3

### DISCUSSION THREAD

**Discussion by tyasnk** [duckdb/duckdb#6659] [Q&A]
Hi,

I try to query from local file using read_parquet and its working. But when i upload it to aws S3 read_parquet from it, i got an error ```Error: Invalid Input Error: Malformed parquet file: sum of total compressed bytes of columns seems incorrect```

![Screenshot 2023-03-10 145531](https://user-images.githubusercontent.com/6858446/224264419-3a61851e-0b3b-4235-b5bb-021ae1649117.png)

**Comment by Mause** [comment]
Can you please raise an issue?
